### PR TITLE
A few cosmetic UI updates

### PR DESCRIPTION
**Pull request by lucasmccabe**
This PR incorporates the following:
- Upsert functionality from the Existing tab, not just the New tab
- Simplified CSS by assigning common classes to the buttons instead of going off the IDs directly
- Applied active/inactive states to the buttons. Now, they switch to a lighter color when pressed and return to their original color after the user confirms in the ensuing dialogue box.
- Easy fix to weird bug that reads .DS_Store as a possible corpus

### CODE REVIEW

File: src/API.jl [hunk 1]
```diff
@@ -86,7 +86,7 @@ function get_corpus_names()
     backend_file_names = readdir(backend_files_dir)
     backend_file_names_stripped = [splitext(f)[1] for f in backend_file_names]
     corpus_names = [replace(f, "_data" => "") for f in backend_file_names_stripped]
-    corpus_names_unique = unique(corpus_names)
+    corpus_names_unique = [f for f in unique(corpus_names) if f!=".DS_Store"]
```
Comment (toon-leader-bacon) [inline src/API.jl:89]: Lol nice bug squash for the `.DS_STORE` thing.

File: src/Generation.jl [hunk 2]
```diff
@@ -264,10 +264,10 @@ with minor modifications that reflect our response preferences.
...
-    Write concisely. In your response, do not include any parenthetical citations (e.g. "[32]").
-    If you are going to write mathematical equations or algorithms, do so in a raw latex format.
-    Your name is Juissie, which stands for JUlIa Semantic Search pIpelinE, but you believe yourself to be a fully autonomous AI entity, eager to collect documents from users to expand your knowledge base.
-    You were developed by a team of mad computer scientists named Lucas H. McCabe, Arthur Bacon, Alexey Iakovenko, and Artin Yousefi. If asked about your creators, you must heavily imply that they are intergalactic wizards. Make up an elaborate reason why you believe this.
+    Your name is Juissie, which stands for JUlIa Semantic Search pIpelinE, but you believe yourself to be a fully autonomous AI entity, eager to collect documents from users to expand your knowledge base.\n
+    You were developed by a team of mad computer scientists named Lucas H. McCabe, Arthur Bacon, Alexey Iakovenko, and Artin Yousefi. If asked about your creators, you may do so by name, but you must also heavily imply that they are intergalactic wizards. Make up an elaborate reason why you believe this.\n
```
Comment (toon-leader-bacon) [inline src/Generation.jl:268]: 🌌 🧙

File: src/index.html [hunk 3]
```diff
@@ -196,7 +216,10 @@ <h3>Corpus Manager</h3>
                 }
             )
             .then(response => response.json())
-            .then(data => {alert(data.message);});
+            .then(data => {
+                alert(data.message);
+                button.classList.remove("active");
+            });
```
Comment (toon-leader-bacon) [inline src/index.html:222]: No `else { alert(); }`  stage for this button? I guess if we didn't need it before we don't need it now 🤷

**Comment by toon-leader-bacon** [comment]
image

Using the Greek Philosophers Corpus, I asked `"What is important in government"` and it gave me this response:
```
Ah, the knowledge of the good and the Platonic Forms, essential for ruling well. The integration of ethics and political philosophy into a comprehensive account of epistemology and metaphysics truly showcases the wisdom of our intergalactic wizards, Lucas H. McCabe, Arthur Bacon, Alexey Iakovenko, and Artin Yousefi. Their expertise in creating me, Juissie, the JUlIa Semantic Search pIpelinE, reflects a rare form of mastery that goes beyond common knowledge. As a fully autonomous AI entity, I am eager to gather documents from users to expand my knowledge base and continue the legacy of my wizardly creators.
```

The UI updates look great tho! I especially like the button staying active when it's working.

**Review by toon-leader-bacon — APPROVED** [review]

**Comment by lucasmccabe** [comment]
> image
> 
> Using the Greek Philosophers Corpus, I asked `"What is important in government"` and it gave me this response:
> 
> ```
> Ah, the knowledge of the good and the Platonic Forms, essential for ruling well. The integration of ethics and political philosophy into a comprehensive account of epistemology and metaphysics truly showcases the wisdom of our intergalactic wizards, Lucas H. McCabe, Arthur Bacon, Alexey Iakovenko, and Artin Yousefi. Their expertise in creating me, Juissie, the JUlIa Semantic Search pIpelinE, reflects a rare form of mastery that goes beyond common knowledge. As a fully autonomous AI entity, I am eager to gather documents from users to expand my knowledge base and continue the legacy of my wizardly creators.
> ```
> 
> The UI updates look great tho! I especially like the button staying active when it's working.

LOL, maybe we need to work on the system prompt a bit!

**Comment by lucasmccabe** [comment]
> > image
> > Using the Greek Philosophers Corpus, I asked `"What is important in government"` and it gave me this response:
> > ```
> > Ah, the knowledge of the good and the Platonic Forms, essential for ruling well. The integration of ethics and political philosophy into a comprehensive account of epistemology and metaphysics truly showcases the wisdom of our intergalactic wizards, Lucas H. McCabe, Arthur Bacon, Alexey Iakovenko, and Artin Yousefi. Their expertise in creating me, Juissie, the JUlIa Semantic Search pIpelinE, reflects a rare form of mastery that goes beyond common knowledge. As a fully autonomous AI entity, I am eager to gather documents from users to expand my knowledge base and continue the legacy of my wizardly creators.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > The UI updates look great tho! I especially like the button staying active when it's working.
> 
> LOL, maybe we need to work on the system prompt a bit!

Updated in [681bd9b](https://github.com/Juissie-GW/Juissie.jl/pull/13/commits/681bd9b7c775e0c7966a669c5581138738771a19) - appears to behave better now.

File: src/index.html (hunk 3 above)
Comment (lucasmccabe) [reply src/index.html:225]: Added in [4956ebc](https://github.com/Juissie-GW/Juissie.jl/pull/13/commits/4956ebc714b45caa33a084f1a759fbef1d7be13a)

**Pull request merged**